docs/doc-guideline.md

```diff
@@ -0,0 +1,52 @@
+# DOC GUIDELINE
+
+## How to write an article for the website
+
+1. All the problems written should necessarily include 3 sections
+   1. Explanation of the article
+   2. Sample input
+   3. Sample output
+
+2. All problems should be supplemented by examples of code. 
+
+3. Every problem should have two or more examples with at least one of them well-explained.
+
+4. Time complexity and space complexity should be included, if applicable.
+
+5. Formulae or any important point should be highlighted.
+
+## GUIDELINES FOR THE CODE
+The guidelines for the code:
+   1.  The width of text in each line should not exceed 60 characters
+
+   2.  All documentations should be written using multi-line comments 
+
+   3. Comments should be written above the code and not beside it.
+
+   4. Function and variable names should follow camelcasing. Example: uglyNumbers()
+
+   5. Indentation should be done using 4 spaces.
+
+   6. In case of the code being written in multiple languages, the output of all the codes should be the same
+
+   7. There should not be any spaces for function call or function declaration
+
+   8. Avoid the use of typedef
```

Comment:
Regarding Code Guidelines have a look at some docs I wrote here: https://docs.google.com/document/d/1WYYc2ymImxS6-TKB6pRX77A9zpXb_ujGjdpfMaTDXwc/edit?usp=sharing

Go over Page 8 and you will see my doc on Code Formatting. You can copy paste stuff here and add more information as per your understanding and observation as well. 

Remember that we are not working on NeoAlgo here, but on Docs. So see things from the perspective of a Content Strategist 😉